tools/release-notes/testdata/test-changelog.md

```diff
@@ -21,3 +27,17 @@ Teleport 16 brings the following new features and improvements:
 - Access requests from the resources view
 - `tctl` for Windows
 - Teleport plugins improvements
+
+## 15.4.17 (08/28/24)
```

Comment:
Perhaps add a test for this version so that there is a test where the changelog is terminated by EOF and not the next heading? Or just remove this section so that the test for v16 does that?